packages/terra-functional-testing/src/services/wdio-visual-regression-service/VisualRegressionLauncher.js

```diff
@@ -29,16 +28,17 @@ class VisualRegressionLauncher {
    * @param {[type]} specs
    * @return null
    */
-  async before() {
+  async before(capabilities) {
     const userAgent = await browser.execute(getUserAgent);
-    const { name, version, ua } = parsePlatform(userAgent);
+    const { name, version, ua } = parse(userAgent);
 
     this.context = {
       browserInfo: {
         name,
         version,
         userAgent: ua,
       },
+      desiredCapabilities: capabilities,
```

Comment:
Since you added `desiredCapabilities` can we remove the `browserInfo` from the context?

Reply:
The reason I added `desireCapabilities` is to get the lower case browser name (e.g. `chrome`) instead of `Chrome` from `browserInfo.name`. Yes, you're right. `browserInfo` was needed only for the browserName so we no longer need to add `browserInfo` to the context. I also removed `getUserAgent` and the `platform` dependency since they were only used to get the user agent data for `browserInfo`.

https://github.com/cerner/terra-toolkit/pull/545/commits/9933d1afc047d7a96ebbc454f3b716a0b8a9a420